src/main/kotlin/xyz/bluspring/kilt/loader/KiltLoader.kt

```diff
@@ -900,6 +940,7 @@ class KiltLoader {
     // We need to initialize all early Forge-related things immediately,
     // because otherwise things will break entirely.
     fun initForge() {
+        runBlocking { scanModJob.join() }
```

Comment:
Why is this repeated here? This job should by all means *never* be executed this late in the mod loading process.

The mod scanning process should already be fully running in `KiltMixinPlugin`, so it can get picked up by `KiltEarlyRiser` later to run the access transformers.

Reply:
See the others